<?xml version="1.0" encoding="UTF-8" ?>
<testsuites errors="2"
            failures="1"
            tests="1">

  <testsuite name="search"
    errors="2" failures="1" hostname="" id="" package="search" skipped="0"
    tests="1" time="5.184" timestamp="Wed, 03 Feb 2016 08:50:16 GMT">
  
    <testcase name="Redirect to Search Page" time="5.184" assertions="3">

    

      <failure message="Failed [equal]: (&#39;http://cummings.localhost/login&#39; == &#39;http://cummings.localhost/#/search&#39;)">Failed [equal]: (&#39;http://cummings.localhost/login&#39; == &#39;http://cummings.localhost/#/search&#39;): Failed [equal]: (&#39;http://cummings.localhost/login&#39; == &#39;http://cummings.localhost/#/search&#39;) - Expected &quot;http://cummings.localhost/#/search&quot; but got: &quot;http://cummings.localhost/login&quot;
    at Object.equal (/usr/local/lib/node_modules/nightwatch/lib/core/api.js:67:24)
    at Object.&lt;anonymous&gt; (/Users/sumangurung/nightwatch_test/tests/search.js:11:15)
    at HttpRequest.&lt;anonymous&gt; (/usr/local/lib/node_modules/nightwatch/lib/index.js:333:20)
    at HttpRequest.emit (events.js:110:17)
    at HttpRequest.&lt;anonymous&gt; (/usr/local/lib/node_modules/nightwatch/lib/index.js:361:15)
    at HttpRequest.emit (events.js:118:17)
    at IncomingMessage.&lt;anonymous&gt; (/usr/local/lib/node_modules/nightwatch/lib/http/request.js:155:16)
    at IncomingMessage.emit (events.js:129:20)
    at _stream_readable.js:908:16
    at process._tickCallback (node.js:355:11)</failure>

    
    </testcase>
  
  
  </testsuite>
</testsuites>
